templates/articles/detail.html

```diff
@@ -1,23 +1,33 @@
 {% extends "base.html" %}
 
+{% block meta_title %}{{ article.title }}{% endblock %}
+{% block meta_description %}{{ article.summary }}{% endblock %}
+
 {% block body %}
-<article>
+<article lang={{ article.language }}>
```

Comment:
Better on the `<html>` element?

Reply:
Would it make sense to have one on HTML and one on the article if the language of the page and the language of the article were different? (more a general question as it may not apply here)

Reply:
If it happens that would be weird for the visitor (as in my blog :p), maybe we should try to avoid that situation with forced translation of the interface too?